packages/collection/src/collection.ts

```diff
@@ -662,6 +662,21 @@ export class Collection<K, V> extends Map<K, V> {
 		return coll;
 	}
 
+	/**
+	 * The missing method returns a new structure containing items where the keys and values of original structure are not present in the other.
+	 *
+	 * @param other - The other Collection to filter against
```

Comment:
nice copy and paste
```suggestion
	 * This method returns a collection containing the items that are missing from the other collection in comparison to the current one
	 *
	 * @param other The other Collection to compare against
```

Reply:
The hyphen separating the parameter name and description should stay
Ref: https://tsdoc.org/pages/tags/param/